apiserver/debuglog.go

```diff
@@ -119,7 +120,7 @@ func (h *debugLogHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		clock := h.ctxt.srv.clock
 		maxDuration := h.ctxt.srv.shared.maxDebugLogDuration()
 
-		if err := h.handle(clock, maxDuration, st, params, socket, h.ctxt.stop()); err != nil {
+		if err := h.handle(clock, maxDuration, st, params, socket, h.ctxt.stop(), st.Removing()); err != nil {
```

Comment:
These channels could be merged together to avoid the `stop2` param, not sure if its really worth it though.

Reply:
Not adding a go routine to just merge two stop channels, the cruft is worth it in this case.